apps/src/javalab/Theater.js

```diff
@@ -1,20 +1,31 @@
-import {TheaterSignalType, STATUS_MESSAGE_PREFIX} from './constants';
+import {
+  TheaterSignalType,
+  STATUS_MESSAGE_PREFIX,
+  InputMessageType
+} from './constants';
 import javalabMsg from '@cdo/javalab/locale';
 
+// Exported for tests
+export const UPLOAD_SUCCESS_MESSAGE = 'UPLOAD_SUCCESS';
```

Comment:
should these go into the constants file instead?

Reply:
Hmm, yeah I wasn't sure since they were messages themselves (not message keys or message types), but it probably does make more sense to put them in constants since they should be consistent with Javabuilder.